be/src/exec/sorting/merge_path.h

```diff
@@ -335,14 +347,20 @@ struct Metrics {
     // 6 -> Stage::FINISHED
     std::vector<RuntimeProfile::Counter*> stage_timers;
     std::vector<RuntimeProfile::Counter*> stage_counters;
+
+    RuntimeProfile::Counter* _provider_timer;
```

Comment:
The counter name is hard understand if user only know sort concept.

Reply:
`provider_timer`->`sorted_run_provider`, what do you think?

Reply:
updated